app/components/project-long-description.js

```diff
@@ -1,13 +1,49 @@
 import Ember from 'ember';
 
+/**
+  `project-long-description` displays and allows editing of the long
+  long description for the project.
```

Comment:
I *think* this is a type `...the long long description.. => ...the long description...`